controllers/books.js

```diff
@@ -0,0 +1,122 @@
+const { Book } = require('../models')
+
+class bookController {
```

Comment:
Nama class biasanya di mulai dengan huruf besar
```suggestion
class BookController {
```